index.js

```diff
@@ -4,7 +4,7 @@ const parseHBS = (node) => {
     return;
   }
   if (
-    /^({{|<[A-Z]|<\/[A-Z]|<[a-zA-Z]+\.[a-zA-Z])/g.test(node.value.trimStart())
```

Comment:
removing this `^` was the fix

Reply:
Just pushed a different approach to fix this issue without removing the `^`.